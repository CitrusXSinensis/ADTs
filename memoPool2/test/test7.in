ra12 12 12 12 23 34 12 23 1 1 2 3 4  2   1
2x
ca
fa 1
aa 1
pa
ca
aa 8
fa 9
fa 7
ca
pa 
