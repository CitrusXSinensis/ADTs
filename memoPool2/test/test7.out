16/16
Chunk: 12
12 12 12 23 34 12 23 1 1 2 3 4 2 1 2 1
16/16
Chunk: 12
Chunk: 12
15/32
12 23 34 12 23 1 1 2 3 4 2 1 2 1 8
